app-vim/pyclewn/pyclewn-2.1-r1.ebuild

```diff
@@ -14,8 +14,8 @@ LICENSE="GPL-2"
 KEYWORDS="~amd64 ~x86"
 
 CDEPEND="|| (
-	>=app-editors/vim-7.3[${PYTHON_USEDEP}]
-	>=app-editors/gvim-7.3[netbeans,${PYTHON_USEDEP}]
+	>=app-editors/vim-7.3
+	>=app-editors/gvim-7.3[netbeans]
```

Comment:
It's said in the commit message. It was unnecessary since this thing does not use vim's python thingy.